apps/full-site-editing/full-site-editing-plugin/full-site-editing/templates/class-wp-template.php

```diff
@@ -161,9 +161,9 @@ public function get_page_template_content() {
 			return null;
 		}
 
-		return "<!-- wp:a8c/template {\"templateId\":$header_id,\"className\":\"site-header site-branding\"} /-->" .
+		return "<!-- wp:a8c/template {\"templateId\":$header_id,\"label\":\"Header\",\"className\":\"site-header site-branding\"} /-->" .
```

Comment:
Here (and later for the footer) we are hardcoding the template part names, while instead we should translate them.

For example:
```php
return '<!-- wp:a8c/template {"label":"' . __( 'Header', 'full-site-editing' ) . '" } /-->';
```

---

In the future, when we start adding proper localized support (or having more template parts with unpredictable names), we might end up replacing the simple translation function with a different approach.
For example, the title of the template part post returned by the API could be already translated, so we could do something lie this:
```php
return '<!-- wp:a8c/template {"label":"' . get_the_title( $header_id ) . '" } /-->';
```

Just to be clear: **this wouldn't work right now**, it's just an idea of what could eventually be.
But now that we have hardcoded template parts, it's plenty enough to just simply translate them.

Reply:
That is great advice, I just added it in the commit as seen below.
Thanks!